go.mod

```diff
@@ -252,7 +252,7 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101411.5-rc.1
+replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101411.5-rc.1.0.20241219170731-928070c7fc09
```

Comment:
pulls in this fix: https://github.com/ethereum-optimism/op-geth/commit/928070c7fc097362ed2d40a4f72889ba91544931, which is not part of a release yet